get-pants.sh

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+function log() {
+  echo -e "$@" >&2
+}
+
+function warn() {
+  log "WARNING: " "$@"
+}
+
+function die() {
+  log "$@"
+  exit 1
+}
+
+function check_cmd() {
+  local cmd="$1"
+  command -v "$cmd" >/dev/null || die "This script requires the ${cmd} binary to be on the PATH."
+}
+
+_GC=()
+
+function gc() {
+  if [[ -z "${NO_GC:-}" ]]; then
+    if (($# > 0)); then
+      check_cmd rm
+      _GC+=("$@")
+    else
+      rm -rf "${_GC[@]}"
+    fi
+  fi
+}
+
+trap gc EXIT
+
+check_cmd uname
+
+function calculate_os() {
+  local os
+
+  os="$(uname -s)"
+  if [[ "${os}" =~ [Ll]inux ]]; then
+    echo linux
+  elif [[ "${os}" =~ [Dd]arwin ]]; then
+    echo macos
+  elif [[ "${os}" =~ [Ww]in|[Mm][Ii][Nn][Gg] ]]; then
+    # Powershell reports something like: Windows_NT
+    # Git bash reports something like: MINGW64_NT-10.0-22621
+    echo windows
+  else
+    die "Integration tests are not supported for this operating system (${os})."
```

Comment:
This message gives me pause.. if this script is run by a user on an unsupported OS, and they see "Integration tests are not supported for this operating system ...", well if they want to get up and running with Pants, what has integration tests got to do with it? (I see that may be the reason why it's not available to them, but I find it a not very helpful message to convey)

Reply:
Copy-pasta: https://github.com/a-scie/jump/blob/main/examples/run.sh
Thanks for catching that.